contracts/deployables/strategies/LinearERC20VotingV1.sol

```diff
@@ -117,7 +117,7 @@ contract LinearERC20VotingV1 is
         governanceToken = IVotes(_governanceToken);
         __Ownable_init(_owner);
         __UUPSUpgradeable_init();
-        _setAzorius(_azoriusModule);
```

Comment:
Something doesn't feels off by this. requiring each strategy to have this line of code for something that is required for the contract to function properly. 

I guess my question comes down to, is there value in those "non 0" 
checks prior to setting the azorius module? if yes, then we should probably keep the internal method and reuse that? 

if no, should we lock all of our strategies to this pattern? we we ever one something else to initial a proposal?

Reply:
Addressed in latest (force pushed) commit